### ISSUE TITLE
Feature Request: Make Pull Request visibility a setting

### ISSUE THREAD

**Issue by tienyuan** [huboard/huboard#630]
I'd like to be able to toggle off Pull Requests and not see them on Huboard.

My team only tracks Issues in our workflow, so our PRs just clog up our first stage.

**Comment by petey-rock** [comment]
Hi Tien,
Here is a short video that demonstrates filtering for issues only. Also, if you copy the url you can share the filtered board with your team. Please let us know if this solves your problem! 
Thanks!

<iframe width="640" height="360" allowTransparency="true" mozallowfullscreen webkitallowfullscreen allowfullscreen style="background-color:transparent;" frameBorder="0" src="https://app.ilosvideos.com/embed/YqBcDrwQVExb"></iframe>

**Comment by petey-rock** [comment]
http://ilos.video/Qxwgot

**Comment by tienyuan** [comment]
Yes, I normally filter now. I'd just rather have this as a optional setting.
